Directory.Packages.props

```diff
@@ -77,7 +77,6 @@
     <PackageVersion Include="RabbitMQ.Client" Version="[6.5.0,7.0)" />
     <PackageVersion Include="StyleCop.Analyzers" Version="[1.2.0-beta.507,2.0)" />
     <PackageVersion Include="Swashbuckle.AspNetCore" Version="[6.4.0]" />
-    <PackageVersion Include="System.Net.Http" Version="4.3.4" />
```

Comment:
This shouldn't be used at all since it's a dead package https://www.nuget.org/packages/System.Net.Http/.